migrations/4_genesis.js

```diff
@@ -8,20 +8,26 @@ const LivepeerToken = artifacts.require("LivepeerToken")
 const GenesisManager = artifacts.require("GenesisManager")
 const TokenDistributionMock = artifacts.require("TokenDistributionMock")
 
-module.exports = function(deployer, network) {
+module.exports = function(deployer, network, accounts) {
     deployer.then(async () => {
         const lpDeployer = new ContractDeployer(deployer, Controller, ManagerProxy)
 
         const controller = await Controller.deployed()
 
         const tokenAddr = await controller.getContract(contractId("LivepeerToken"))
-        const faucetAddr = await controller.getContract(contractId("LivepeerTokenFaucet"))
         const minterAddr = await controller.getContract(contractId("Minter"))
 
         const token = await LivepeerToken.at(tokenAddr)
 
-        const tokenDistribution = await lpDeployer.deploy(TokenDistributionMock, tokenAddr, faucetAddr, genesis.tokenDistribution.endTime)
-        const genesisManager = await lpDeployer.deploy(GenesisManager, tokenAddr, tokenDistribution.address, genesis.bankMultisig, minterAddr)
+        const dummyTokenDistribution = await lpDeployer.deploy(TokenDistributionMock, genesis.dummyTokenDistribution.timeToEnd)
+
+        let genesisManager
+
+        if (!lpDeployer.isLiveNetwork(network)) {
+            genesisManager = await lpDeployer.deploy(GenesisManager, tokenAddr, dummyTokenDistribution.address, accounts[0], minterAddr)
```

Comment:
If we're not deploying to a live network, transfer the "company supply" to the account used for the deployment transactions so that account can transfer LPT to other accounts during automated tests